crates/nu_plugin_polars/src/dataframe/values/nu_expression/custom_value.rs

```diff
@@ -133,6 +133,21 @@ fn with_operator(
             .apply_with_expr(right.clone(), Expr::lt_eq)
             .cache(plugin, engine, lhs_span)?
             .into_value(lhs_span)),
+        Operator::Boolean(Boolean::And) => Ok(left
```

Comment:
The custom_value.rs file probably shouldn't be here at all since this PR just talks about plugin documentation

Reply:
It is only part of this PR because it depends on #15249 and on #15248 . Once these are in main this file shouldn't show up anymore. The dependence is there because the doc mentions Boolean operators which otherwise wouldn't exist.